examples/quickstart/jupyter-notebooks/notebooks/03-query/02-approxRanking.ipynb

```diff
@@ -0,0 +1,577 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f388633f-195b-4381-98cc-7a2f80f48690",
+   "metadata": {},
+   "source": [
+    "## Example TopN style queries\n",
+    "\n",
+    "Druid looks for patterns in `SELECT` statements to determine if they would benefit from using approximation. A ranking query, like the one below, matches the rules for TopN approximation, so Druid enables it by default.\n",
+    "\n",
+    "To see this happen, we need an SQL statement that has:\n",
+    "* A GROUP BY on one dimension, and\n",
+    "* an ORDER BY on one aggregate.\n",
+    "\n",
+    "Run this query to see what the results are like:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b76e5184-9fe4-4f21-a471-4e15d16515c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = '''\n",
+    "SELECT\n",
+    "    \"Reporting_Airline\",\n",
+    "    COUNT(*) AS Flights,\n",
+    "    SUM(\"Distance\") AS SumDistance\n",
+    "FROM\n",
+    "    \"example-flights-topn\"\n",
+    "GROUP BY 1\n",
+    "ORDER BY 2 DESC\n",
+    "LIMIT 10\n",
+    "'''\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5600fc48-c999-406f-800b-3f0f6a973aa0",
+   "metadata": {},
+   "source": [
+    "You can use `EXPLAIN PLAN FOR` or the `explain_sql` method to see whether Druid used TopN approximation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7595eec0-a709-4cd6-985e-eec8a6e37b61",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(json.dumps(json.loads(sql_client.explain_sql(sql)['PLAN']), indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8658b26e-2f09-4a97-96e8-589168130559",
+   "metadata": {},
+   "source": [
+    "You know approximation is used when the `queryType` is `topN`.\n",
+    "\n",
+    "For these queries, Druid applies a `LIMIT` operation on the results calculated by each server that’s been called upon to answer the query, not just on the final result set. This sees less data bubbling up from each process to be merged overall, and therefore greater efficiency in execution.\n",
+    "\n",
+    "There's an important reason why our query doesn't include a `HAVING` clause: for `HAVING` to work properly, Druid needs to have the full and final results of your aggregations. A single data service doesn't have the full picture - that's only available after all the results are merged.\n",
+    "\n",
+    "Notice the `threshold` value?\n",
+    "\n",
+    "```json\n",
+    "    \"threshold\": 10,\n",
+    "```\n",
+    "\n",
+    "The parallelised `LIMIT` was the `max` of both the `threshold` shown here – which came from the `LIMIT` in the SQL - and a configuration setting in your cluster – the default for which is 1,000.\n",
+    "\n",
+    "You can find out how to read and set this default `LIMIT` in the [documentation](https://druid.apache.org/docs/latest/querying/topnquery.html#aliasing).\n",
```

Comment:
It seems like what follows is going to demonstrate an instance where we hit the limit? I'm not sure otherwise why I might need to change it.